app/src/main/java/com/ciandt/dragonfly/example/infrastructure/extensions/Int+Extension.kt

```diff
@@ -0,0 +1,7 @@
+package com.ciandt.dragonfly.example.infrastructure.extensions
+
+import android.content.res.Resources
+
+fun Int.toPx(): Int = (this * Resources.getSystem().displayMetrics.density).toInt()
```

Comment:
Arguments:

- Best semantics
1.px() sounds like 1px, and not 1dp converted to px

- Keep the pattern
```
fun toByte(): Byte
fun toChar(): Char
fun toShort(): Short
fun toInt(): Int
fun toLong(): Long
fun toFloat(): Float
fun toDouble(): Double
```

makes sense?